examples/animation/easing_functions.rs

```diff
@@ -16,7 +16,7 @@ fn main() {
 fn setup(mut commands: Commands) {
     commands.spawn(Camera2d);
 
-    let text_style = TextStyle {
+    let text_style = TextFont {
```

Comment:
Just for sake of understanding, can we also change this var name to text_font? Probably will be more understandable this way.

Reply:
Yep missed the one,